.eslintrc.js

```diff
@@ -46,34 +46,6 @@ module.exports = {
         '@typescript-eslint/no-explicit-any': 'off',
         'lines-between-class-members': ['error', 'always', {exceptAfterSingleLine: true}],
         'tsdoc/syntax': 'off', // Todo: remove this before merging pr
-        'valid-jsdoc': [
```

Comment:
As discussed previously as well and suggested from Marcin, we don't want to add jsdoc in ts files so that's why removed this eslint check for jsdoc in ts files. Will add a commit which will remove all existing jsdocs also from calling folder.